src/main/webapp/js/stats-loader.js

```diff
@@ -0,0 +1,23 @@
+// Fetch stats and display them in the page.
+function fetchStats(){
+    const url = '/stats';
+    fetch(url).then((response) => {
+      return response.json();
+    }).then((stats) => {
+      const statsContainer = document.getElementById('stats-container');
+      statsContainer.innerHTML = '';
+      const messageCountElement = buildStatElement('Message count: ' + stats.messageCount);
+      statsContainer.appendChild(messageCountElement);
+    });
+  }
+
+  function buildStatElement(statString){
+   const statElement = document.createElement('p');
+   statElement.appendChild(document.createTextNode(statString));
+   return statElement;
+  }
+
+  // Fetch data and populate the UI of the page.
+  function buildUI(){
```

Comment:
Is there a reason that you have an alias like this? Why not just call it using the `fetchStats` function name and remove this alias?

Reply:
For this instance, there is no reason. I was thinking that as we add more things to the stats page, that having  a general function "buildUI" is better than having the html page calling all of the different functions

Reply:
Let's follow the "YAGNI" (you ain't gonna need it) principle; I concur that it would make sense for the HTML code to call only a single function to populate the entire UI, but let's start with it just calling the one function (without an alias); if you decide that you need multiple functions, you can always introduce this layer of indirection when it becomes necessary.